feat: es translations for New Maintainers course

samucodesh opened the pull request:
This PR contains the translation of 5 files from English to Spanish for the maintainers course, in a few hours I will add another 3 files

feat: Translate Introduction to Spanish
feat: Translate Maintainer Power-ups to Spanish
feat: Translate How to Setup Your Project to Spanish
fix: Correct filename for 'Issues and Pull Requests' in Spanish
feat: Translate Issues and Pull Requests to Spanish
feat: Translate Your Team to Spanish

samucodesh committed:
feat: Add Spanish translations for Maintainer Power Ups

- Added a new folder called translations and within it, another folder called es. Inside the es folder, I've added the Spanish translation for the Maintainer Power Ups markdown file.

samucodesh committed:
Merge branch 'open-sauced:main' into a-ridley/intro

samucodesh committed:
feat: Translate Maintainer Power-ups to Spanish

This commit translates the original Maintainer Power-ups (maintainer-powerups.md) into Spanish, creating a new file named potenciadores-para-mantenedores.md. 

The new file includes the following topics translated into Spanish:

- GitHub Actions
- CI/CD Pipelines
- GitHub CLI
- Issues and Pull Request Templates
- Saved Replies
- Code Owners

samucodesh committed:
feat: Translate Introduction to Spanish

This commit translates the original Introduction file (intro.md) into Spanish, creating a new file named introduccion.md.

The new file includes the following topics translated into Spanish:
    
    - What is an Open Source Maintainer?
    - Benefits of being a maintainer
    - Challenges of being a maintainer
    - Key Responsibilities and Expectations

samucodesh committed:
feat: Translate How to Setup Your Project to Spanish

    This commit translates the original How to Setup Your Project file (how-to-setup-your-project.md) into Spanish, creating a new file named como-configurar-tu-proyecto.md.

    The new file includes the following topics translated into Spanish:

    - Detailed README
    - Installation Guide
    - Open Source Software License
    - Code of Conduct
    - Contributing Guidelines
    - Issue Templates
    - Pull Request Templates
    - Projects on GitHub

samucodesh committed:
fix: Correct filename for 'Issues and Pull Requests' in Spanish

This commit fixes a typo in the filename for the Spanish translation of the 'Issues and Pull Requests'

samucodesh committed:
feat: Translate Issues and Pull Requests to Spanish

    This commit translates the original Issues and Pull Requests file (issues-and-pull-requests.md) into Spanish, creating a new file named problemas-y-solicitudes-de-extraccion.md.

    The new file includes the following topics translated into Spanish:

    - Prioritizing and Managing Issues
    - Reviewing Pull Requests
    - Highlights for Issues and Pull Requests with OpenSauced
    - Respond Promptly and Address Concerns
    - Prioritizing Tasks and Realistic Timelines
    - Handling Security Vulnerabilities

samucodesh committed:
feat: Translate Your Team to Spanish

    This commit translates the original Your Team file (your-team.md) into Spanish, creating a new file named tu-equipo.md.

    The new file includes the following topics translated into Spanish:

    - Types of Teams
    - Building Your A-Team: Adding New Members
    - Granting Team Permissions
    - Expanding Horizons: Enlisting Additional Maintainers
    - Adding Team Members
    - Onboarding New Team Members
    - Keeping Track of Your Team
    - Saying Goodbye: Handling Team Departures
    - Conclusion

samucodesh committed:
feat: Translate Metrics and Analytics to Spanish

    This commit translates the original Metrics and Analytics file (metrics-and-analytics.md) into Spanish, creating a new file named metricas-y-analisis.md.

    The new file includes the following topics translated into Spanish:

    - Understanding Types of Metrics
    - Setting Project Goals
    - Understanding and Interpreting Open Source Project Data
    - Making Data-Driven Decisions
    - Creating a Workspace for Your Project with OpenSauced
    - Leveraging OpenSauced for Project Success

samucodesh committed:
feat: Translate Maintaining Code Quality to Spanish

    This commit translates the original Maintaining Code Quality file (maintaining-code-quality.md) into Spanish, creating a new file named manteniendo-la-calidad-del-codigo.md.

    The new file includes the following topics translated into Spanish:

    - Code Reviews and High-Quality Contributions
    - Automated Testing
    - Code Quality Checks
    - Branch Protection Rules
    - Maintaining Project Documentation

samucodesh committed:
feat: Translate Maintainers Guestbook to Spanish

    This commit translates the original Maintainers Guestbook file (maintainers-guestbook.md) into Spanish, creating a new file named libro-de-visitas-de-mantenedores.md.

    The new file includes a section for listing projects by maintainers.

samucodesh committed:
update: Add Maintainer Guestbook entries

[Related PR](https://github.com/open-sauced/intro/pull/200)

    This commit adds two new entries to the Maintainer Guestbook:

    - Virtual Coffee by VirtualCoffee
    - Postpartum Wellness App by BekahHW

a-ridley commented on docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md:
```diff
@@ -0,0 +1,208 @@
+# Cómo configurar tu proyecto Open Source
```
Consider using the following instead 

"Cómo configurar tu proyecto de Open Source"

a-ridley commented on docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md:
```diff
@@ -0,0 +1,208 @@
+# Cómo configurar tu proyecto Open Source
+
+Hay muchas cosas a considerar al configurar tu proyecto Open Source. En esta parte del curso, revisaremos los componentes clave que todo proyecto necesita para tener éxito.
```
consider changing "Hay muchas cosas a considerar al configurar tu proyecto Open Source.

to: "Hay muchas cosas a considerar al configurar tu proyecto **de** Open Source."

and

consider changing:  "En esta parte del curso, revisaremos los componentes clave que todo proyecto necesita para tener éxito."

to: "En esta parte del curso, revisaremos los componentes clave que **todos los proyectos necesitan** para tener éxito.

a-ridley commented on docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md:
```diff
@@ -0,0 +1,208 @@
+
+## README detallado
+
+Un archivo README contiene una introducción a tu proyecto Open Source. Un buen README debe ser claro, conciso, actualizado y detallado. Este archivo se encuentra en tu directorio raíz y sirve como la primera impresión para tus colaboradores.
```
Consider changing "Un archivo README contiene una introducción a tu proyecto Open Source." 

to: "Un archivo README contiene una introducción a tu proyecto **de** Open Source."

a-ridley commented on docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md:
```diff
@@ -0,0 +1,208 @@
+Para crear plantillas de problema usando el generador de plantillas de GitHub, deberás:
+
+- Ir a la configuración de tu proyecto,
+- Navegar a la sección "Características",
```
I feel like instructions that are for navigating Github should be kept in english since the website itself has those functions in english.

That would mean changing:
- Navegar a la sección "Características"

to: - Navegar a la sección "Features"

a-ridley commented on docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md:
```diff
@@ -0,0 +1,208 @@
+
+- Ir a la configuración de tu proyecto,
+- Navegar a la sección "Características",
+- Haz clic en "Configurar plantillas" debajo de "Problemas".
```
Similar to above, this is an instruction for navigating GitHub which has these buttons in english. 

- Haz clic en "Set up templates" debajo de "Issues".

a-ridley commented on docs/becoming-a-maintainer/translations/es/como-configurar-tu-proyecto.md:
```diff
@@ -0,0 +1,208 @@
+
+#### 2. Usando archivos YAML
+
+También puedes crear plantillas de problema personalizadas usando la sintaxis de  YAML:
```
```suggestion
También puedes crear plantillas de problema personalizadas usando la sintaxis de YAML:
```

a-ridley commented on docs/becoming-a-maintainer/translations/es/libro-de-visitas-de-mantenedores.md:
```diff
@@ -0,0 +1,13 @@
+# Libro de visitas de  mantenedores
```
```suggestion
# Libro de visitas de mantenedores
```

a-ridley commented on docs/becoming-a-maintainer/translations/es/libro-de-visitas-de-mantenedores.md:
```diff
@@ -0,0 +1,13 @@
+
+## Lista de proyectos
+
+<!-- Use the format below to list your repository -->
```
Did you miss translating this comment? or is this intentional and you are not translating comments? :)

samucodesh replied:
Hello Abi, that is correct, the comments should not be translated.

a-ridley commented on docs/becoming-a-maintainer/translations/es/potenciadores-para-mantenedores.md:
```diff
@@ -0,0 +1,140 @@
+
+Aquí tienes algunos recursos adicionales para ofrecerte más información sobre las GitHub Actions y cómo configurar una:
+
+- [GitHub Actions: El mejor amigo de un mantenedor](https://dev.to/opensauced/github-actions-a-maintainers-best-friend-488n)
```
Instead of GitHub Actions consider using Acciones de GitHub as it is part of the glossary

a-ridley commented on docs/becoming-a-maintainer/translations/es/potenciadores-para-mantenedores.md:
```diff
@@ -0,0 +1,140 @@
+A continuación, se presentan algunos recursos útiles para ayudarte a construir un Pipeline de CI/CD con GitHub Actions:
+
+- [Documentación de GitHub: La solución CI/CD completa](https://github.com/solutions/ci-cd/)
+- [Cómo construir un Pipeline de CI/CD con GitHub Actions en cuatro sencillos pasos](https://github.blog/2022-02-02-build-ci-cd-pipeline-github-actions-four-steps/)
```
replace Github Actions with Acciones de GitHub this will ensure there is consistency :)

a-ridley commented on docs/becoming-a-maintainer/translations/es/problemas-y-solicitudes-de-extraccion.md:
```diff
@@ -0,0 +1,181 @@
+
+Un error que no afecta la funcionalidad de tu aplicación no se considera crítico. Puedes categorizar los errores como de nivel medio cuando puedes corregirlos después de desplegar un lanzamiento próximo. Aquellos que puedas o no incluir en el desarrollo se consideran errores pequeños.
+
+Puedes abrir estos problemas para que cualquiera trabaje en ellos. Algunos de estos problemas pueden ser tan pequeños que serían una buena oportunidad para un nuevo colaborador. Etiqueta el problema con un `buen primer problema` o `solo para principiantes` en esta situación.
```
`buen primer problema`
`solo para principiantes`

Should we keep these labels in english?

`good first issues` 
`first timers only`

I noticed in another file we kepted the labels `Needs triage` and `Bug` in english so figured we should stay consistent. :)

a-ridley commented on docs/becoming-a-maintainer/translations/es/problemas-y-solicitudes-de-extraccion.md:
```diff
@@ -0,0 +1,181 @@
+
+### Problemas etiquetados como características
+
+Al priorizar las solicitudes de características, debes asegurarte de que sean adecuadas para tu proyecto y que te interese agregarlas. Comunícate con el autor original del problema para ver si quiere trabajar en él. Si están de acuerdo, continúa y asígnale ese problema. De lo contrario, agrega una etiqueta `se necesita ayuda` o `aceptando PR`.
```
similar comment about whether we want to keep all referenced labels in english.

`se necesita ayuda` to `help wanted` 
and `aceptando PR` to `accepting PRs`

a-ridley commented on docs/becoming-a-maintainer/translations/es/problemas-y-solicitudes-de-extraccion.md:
```diff
@@ -0,0 +1,181 @@
+
+### Problemas etiquetados como documentación
+
+Si el problema tiene que ver con errores tipográficos o gramaticales menores, agrega una etiqueta `buen primer problema` para abrirlo a nuevos colaboradores. Si el problema es más complejo, deberás evaluar el nivel de prioridad y agregar una etiqueta `se necesita ayuda`. Además, en estas situaciones, es posible que debas obtener más información del autor original antes de abrirlo para la contribu ...
```
similar comment about needing to decide if we want to keep all labels in english or translate. It looks like in this file all labels have been translated so maybe thats what we have decided on.

a-ridley commented on docs/intro-to-oss/how-to-contribute-to-open-source.md:
These are changes to the english version of the intro-to-oss course can we limit this PR to only Spanish translations for the New Maintainers course?

a-ridley replied:
Nvm needed to rebase with main :) ill resolve the comments.

a-ridley commented on docs/intro-to-oss/intro.md:
These are changes to the english version of intro-to-oss can we limit this PR to only Spanish translations for the New Maintainers course?

a-ridley commented on docs/intro-to-oss/translations/fr/_sidebar.md:
This is adding a logo to the intro-to-oss course for the french translation. This should be in a separate PR to the main repo that is maintained by the core opensauced team.

a-ridley reviewed (changes requested):
This is really great work ! 🚀 I read through all the files. I left a few suggestions looking forward to the next iteration of this PR. :) 

Te recap: 

Some of the questions i had while reviewing was consistency with translating PR labels. For the majority of the time all labels were translated but I think there were a few times were some labels were kept in english. Let's stay consistent. :) 

Something else I noticed is sometiems 'de' was used and sometimes it wasn't. 
For example "proytecto Open Source' vs "projecto de Open sourced' let's also stay consistent. I am okay with either one although having the de to me feels cleaner.

samucodesh committed:
fix: Maintain consistency in Spanish translation

    This commit fixes an inconsistency in the Spanish translation of 'proyecto Open Source' to 'proyecto de Open Source' within the 'Manteniendo la calidad del código' file for consistency.

samucodesh committed:
fix: Maintain consistency in Spanish translation

    This commit fixes inconsistencies in the Spanish translation of the 'Cómo configurar tu proyecto Open Source' file for consistency:

    -  'proyecto Open Source' is changed to 'proyecto de Open Source'
    - 'Características' is changed to 'Features'
    - 'Configurar plantillas' is changed to 'Set up templates'
    - 'Problemas' is changed to 'Issues'

samucodesh committed:
fix: Maintain consistency in Spanish translation

    This commit fixes an inconsistency in the Spanish translation of 'proyecto Open Source' to 'proyecto de Open Source' within the 'Introducción' file for consistency.

samucodesh committed:
fix: Fix spacing in Maintainers Guestbook title

    This commit fixes the spacing in the title of the Maintainers Guestbook section in the Spanish translation.

samucodesh committed:
fix: Fix spacing in YAML section

    This commit fixes the spacing before 'YAML' in the 'Cómo configurar tu proyecto' file for consistency.

samucodesh committed:
fix: Maintain consistency in Spanish translation of 'GitHub Actions'

    [Related Issue](https://github.com/a-ridley/intro/pull/1)

    This commit fixes an inconsistency in the Spanish translation of 'GitHub Actions' to 'Acciones de GitHub' within the 'Potenciadores para Mantenedores' file for consistency.

samucodesh committed:
fix: Update Spanish translations for consistency and clarity

    [Related Issue](https://github.com/a-ridley/intro/pull/1)

    This commit updates the Spanish translation of the 'Problemas y Solicitudes de Extracción' file for consistency and clarity:

    - Changed GitHub tag translations to maintain consistency across the project, including 'good first issues', 'help wanted', and 'accepting PRs'.
    - Removes unnecessary spacing in some sentences.

samucodesh committed:
fix: Correct link to 'Potenciadores para Mantenedores' section

    [Related Issue](https://github.com/a-ridley/intro/pull/1)

    This commit fixes a broken link in the 'Manteniendo la calidad del código' file, which was incorrectly referencing the 'Potenciadores para Mantenedores' section. This ensures that the links within the document are accurate and function as expected.

samucodesh committed:
fix: Maintain consistency in Spanish translation of 'Proyecto Open Source'

    [Related Issue](https://github.com/a-ridley/intro/pull/1)

    This commit fixes an inconsistency in the Spanish translation of 'Proyecto Open Source' to 'Proyecto de Open Source' within the 'Problemas y Solicitudes de Extracción' file for consistency.

samucodesh committed:
fix: Maintain consistency in Spanish translation of 'Proyecto Open Source'

    [Related Issue](https://github.com/a-ridley/intro/pull/1)

    This commit fixes an inconsistency in the Spanish translation of 'Proyecto Open Source' to 'Proyecto de Open Source' within the 'Tu Equipo' file for consistency.

samucodesh committed:
fix: Maintain consistency in Spanish translation of 'Proyecto Open Source'

    [Related Issue](https://github.com/a-ridley/intro/pull/1)

    This commit fixes an inconsistency in the Spanish translation of 'Proyecto Open Source' to 'Proyecto de Open Source' within the 'Métricas y Análisis' file for consistency.

samucodesh committed:
fix: Correct link to 'Potenciadores para Mantenedores' section

[Related Issue](https://github.com/a-ridley/intro/pull/1)

 This commit fixes a broken link in the 'Manteniendo la calidad del código' file
